warp/src/main/java/com/schibsted/nmp/warp/components/legacy/WarpRadioView.kt

```diff
@@ -0,0 +1,59 @@
+package com.schibsted.nmp.warp.components.legacy
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.AbstractComposeView
+import androidx.core.content.withStyledAttributes
+import com.schibsted.nmp.warp.R
+import com.schibsted.nmp.warp.components.WarpRadio
+import com.schibsted.nmp.warp.components.ext.getTextFromIdOrString
+
+class WarpRadioView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : AbstractComposeView(context, attrs, defStyle) {
+
+    var text: String = ""
+        set(value) {
+            field = value
+            disposeComposition()
+        }
+
+    var radioSelected: Boolean = false
+        set(value) {
+            field = value
+            disposeComposition()
+        }
+
+    var radioEnabled: Boolean = true
+        set(value) {
+            field = value
+            disposeComposition()
+        }
+
+    private var clickListener = OnClickListener { }
+
+    override fun setOnClickListener(onClick: OnClickListener?) {
+        onClick?.let { clickListener = it }
+    }
+
+    init {
+        context.withStyledAttributes(attrs, R.styleable.WarpRadio) {
+            text = getTextFromIdOrString(R.styleable.WarpRadio_radioText, context) ?: ""
+            radioSelected = getBoolean(R.styleable.WarpRadio_radioSelected, false)
+            radioEnabled = getBoolean(R.styleable.WarpRadio_radioEnabled, true)
+        }
+    }
+
+    @Composable
+    override fun Content() {
+        WarpRadio(
```

Comment:
Wrap the component with the theme to get correct brand colors.
`theme { ..... }`